src/Glpi/Asset/AssetDefinition.php

```diff
@@ -478,12 +478,31 @@ private function prepareInput(array $input): array|bool
 
     public function post_addItem()
     {
+        /** @var \DBmysql $DB */
+        global $DB;
+
         if ($this->isActive()) {
             $this->syncProfilesRights();
 
             // Force menu refresh when active state change
             unset($_SESSION['menu']);
         }
+
+        // Add default display preferences for the new asset definition
+        // Status, Manufacturer, Serial, Type, Model, Location, Last Update
+        $prefs = [31, 23, 5, 4, 40, 3, 19];
+        $rank = 1;
+        foreach ($prefs as $field) {
+            $DB->insert(
```

Comment:
I think it should be done using the `DisplayPreference::add()` method. I know most of operations in `DisplayPreference` class are done directly with low level DB methods, but IMHO, it is an issue as it prevent plugins to be able to use hooks on these operations.